setup.py

```diff
@@ -46,6 +46,7 @@
     author='The Ethereum Foundation',
     author_email='snakecharmers@ethereum.org',
     url='https://github.com/ethereum/eth-account',
+    package_data={'eth_account': ['hdaccount/wordlist/*.txt']},
```

Comment:
I think we have typically done this using a `MANIFEST.in` file.

Reply:
Do you have an example? I'm not sure what you mean

Reply:
https://github.com/ethpm/ethpm-cli/blob/fb4c666a804119cf24390c740ed3e6f472f87bd0/MANIFEST.in